homeassistant/components/tedee/coordinator.py

```diff
@@ -83,3 +83,12 @@ async def _async_update_locks(
             raise UpdateFailed("Error while updating data: %s" % str(ex)) from ex
         except (TedeeClientException, TimeoutError) as ex:
             raise UpdateFailed("Querying API failed. Error: %s" % str(ex)) from ex
+
+    async def async_config_entry_first_refresh(self) -> None:
+        """Update data from API."""
+        await super().async_config_entry_first_refresh()
+
+        async def _async_get_bridge() -> None:
+            self.bridge = await self.tedee_client.get_local_bridge()
+
+        await self._async_update(_async_get_bridge)
```

Comment:
Hmm, not sure if I completely like that we wrap around this. Can't we check in the `_async_update_data` if `self.bridge` is `None` and then execute this function?